src/services/test-helpers/responses/blocks/block789629Extrinsic.json

```diff
@@ -0,0 +1,37 @@
+{
+    "at": {
+        "number": "789629",
+        "hash": "0x7b713de604a99857f6c25eacc115a4f28d2611a23d9ddff99ab0e4f1c17a8578"
+    },
+    "extrinsics": {
+        "method": {
+            "pallet": "timestamp",
+            "method": "set"
+        },
+        "signature": null,
+        "nonce": null,
+        "args": {
+            "now": "1595260344000"
+        },
+        "tip": null,
+        "hash": "0xeb76fff97b8ec4e6b309a266728be3e216abd269fc2e6ee6434f7cce74be515e",
+        "info": {},
+        "events": [
+            {
+                "method": {
+                    "pallet": "system",
+                    "method": "ExtrinsicSuccess"
+                },
+                "data": [
+                    {
+                        "weight": "158000000",
+                        "class": "Mandatory",
+                        "paysFee": "Yes"
+                    }
+                ]
+            }
+        ],
+        "success": true,
+        "paysFee": false
+    }
+}
```

Comment:
Most editors can be configured to ensure there is a newline ending each file.

Reply:
Interesting, i thought the linter would catch that. I'll make sure to configure my editor as well.

Reply:
I actually don't know why it's being signaled by github. Like why is not having a final LF a concern anyway? But there it is and my eye is drawn to "red thingies on screen", so might as well obey. :)